CONTRIBUTING.md

```diff
@@ -164,12 +168,43 @@ Enhancement suggestions are tracked as [GitHub issues](https://github.com/semiot
 
 <!-- You might want to create an issue template for enhancement suggestions that can be used as a guide and that defines the structure of the information to be included. If you do so, reference it here in the description. -->
 
-### Improving The Documentation
-<!-- TODO
-Updating, improving and correcting the documentation
-
--->
-TODO
+### Contributing PRs
```

Comment:
Do you want to attribute these to Colprac to be safe? Make it obvious where you got these from

Reply:
https://github.com/semiotic-ai/autoagora-agents/pull/1/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R248 not enough?

Reply:
Didn't see that! Thanks!